packages/postcss-merge-longhand/src/__tests__/boxBase.js

```diff
@@ -119,16 +119,16 @@ addTests({
     expected: 'h1{box-bottom:var(--variable)}',
 }, {
     message: 'should not merge box props where one has an unset property',
-    fixture: 'h1{box-bottom:10px;box-top:unset;box-left:20px}',
-    expected: 'h1{box-bottom:10px;box-top:unset;box-left:20px}',
+    fixture: 'h1{box-bottom:10px;box-top:unset;box-left:20px;box-right:20px}',
+    expected: 'h1{box-bottom:10px;box-top:unset;box-left:20px;box-right:20px}',
 }, {
     message: 'should not merge box props where one has an initial property',
-    fixture: 'h1{box-bottom:10px;box-top:initial;box-left:20px}',
-    expected: 'h1{box-bottom:10px;box-top:initial;box-left:20px}',
+    fixture: 'h1{box-bottom:10px;box-top:initial;box-left:20px;box-right:20px}',
+    expected: 'h1{box-bottom:10px;box-top:initial;box-left:20px;box-right:20px}',
 }, {
     message: 'should not merge box props where one has an inherit property',
-    fixture: 'h1{box-bottom:10px;box-top:initial;box-left:20px}',
-    expected: 'h1{box-bottom:10px;box-top:initial;box-left:20px}',
+    fixture: 'h1{box-bottom:10px;box-top:initial;box-left:20px;box-right:20px}',
+    expected: 'h1{box-bottom:10px;box-top:initial;box-left:20px;box-right:20px}',
```

Comment:
Please add new tests, don't change exists, it is bad practice :+1: This can lead to regressions in unexpected places.

Reply:
These are probably duplicates of existing tests (checking for incomplete box props) hence why they did not increase coverage in the previous PR. However I appreciate the caution and have kept them the same but renamed the message to better match the area they are touching. Hope that's OK.